wiki/math/registration-techniques.md

```diff
@@ -1,110 +1,109 @@
 * The robot is fitted with a colored bead on its end effector; the colored bead can be easily segmented from its background by hue, saturation, and value. It is preferred that the bead be bright in color so as to increase its visibility to the stereo camera.
 * The robot is moved to a fixed set of five or six points. The points are chosen to cover a significant amount of the robot's workspace and also stay within the field of view in the camera. 
 * The robot is moved to the specified location. The left and right images of the stereo are processed to find the centroid of the colored bead fitted to the robot. This is repeated for as many frames as are received over ROS (which is the framework used in [5]) in one second. The centroid is averaged over all the received frames for a more accurate spatial estimate of the bead.
 * The pixel disparity is given as input to a stereo-camera model that ROS provides to calculate a 3D point in the camera-frame.
 * Following this, we have six points in both the camera-frame and the robot-frame. The coordinates of the point in the robot-frame are known using the kinematics of the robot.
-* Since we now have corresponding points in the camera-frame and the robot-frame, Horn's method is used to calculate the transformation <a href="https://www.codecogs.com/eqnedit.php?latex=$T_r^c$" target="_blank"><img src="https://latex.codecogs.com/gif.latex?$T_r^c$" title="$T_r^c$" /></a> between the robot-frame and the camera-frame.
+* Since we now have corresponding points in the camera-frame and the robot-frame, Horn's method is used to calculate the transformation \\[ T_r^c \\] between the robot-frame and the camera-frame.
 
 ### Iterative Closest Point Algorithm
 
-Iterative Closest Point (ICP) is an algorithm to register two sets of point clouds iteratively. In each iteration, the algorithm selects the closest points as correspondences and calculates the transformation, i.e., rotation and translation (<a href="https://www.codecogs.com/eqnedit.php?latex=R" target="_blank"><img src="https://latex.codecogs.com/gif.latex?R" title="R" /></a> and <a href="https://www.codecogs.com/eqnedit.php?latex=t" target="_blank"><img src="https://latex.codecogs.com/gif.latex?t" title="t" /></a>). Although there are a number of techniques to calculate the transformation at each iteration, Horn's method remains a popular choice to do so. 
+Iterative Closest Point (ICP) is an algorithm to register two sets of point clouds iteratively. In each iteration, the algorithm selects the closest points as correspondences and calculates the transformation, i.e., rotation and translation (\\[ R \\] and \\[ t \\]. Although there are a number of techniques to calculate the transformation at each iteration, Horn's method remains a popular choice to do so. The figure below illustrates the registration of two different sets of point clouds(shown in pink and blue for distinction).
+
+TODO ADD IMAGE
```

Comment:
Do you have something to add here? You can add it to the assets folder and link to it from there.